Change MessageType values from decimals to hexadecimals for readability

pbelevich opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#55985 Change MessageType values from decimals to hexadecimals for readability**

Differential Revision: [D27758101](https://our.internmc.facebook.com/intern/diff/D27758101)

pbelevich committed:
Change MessageType values from decimals to hexadecimals for readability

[ghstack-poisoned]

mrshenli reviewed (approved):
LGTM. Thanks for improving!

pbelevich committed:
Update on "Change MessageType values from decimals to hexadecimals for readability"


Differential Revision: [D27758101](https://our.internmc.facebook.com/intern/diff/D27758101)

[ghstack-poisoned]
